src/core/ConfigManager.cpp

```diff
@@ -131,17 +133,17 @@ void ConfigManager::upgrade()
 		return;
 	}
 
-	ProjectVersion createdWith = m_version;
-	
-	if (createdWith.setCompareType(ProjectVersion::Build) < "1.1.90")
+	if( configVersion() < 1 )
```

Comment:
As suggested by @Veratil on Discord, we can use a switch fallthrough here instead:
```suggestion
	switch (configVersion()) {
		case 0: upgrade_0();
		case 1: upgrade_1();
	}
```